app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftConfigurationProperties.java

```diff
@@ -138,4 +141,20 @@ public String getIntegrationDataPath() {
     public void setIntegrationDataPath(String integrationDataPath) {
         this.integrationDataPath = integrationDataPath;
     }
+
+    public int getMaximumRetries() {
+        return maximumRetries;
+    }
+
+    public void setMaximumRetries(int maximumRetries) {
```

Comment:
@iocanel do you want to set an examples already in the application.yml (with some comments and the defaults), so that people know that they can tune it here ?

If you don't have time or don't feel like it, I can add this, too.